app/assets/javascripts/modules/toggle-class.js

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
As discussed offline, I think this would be simpler re-producing just the module behaviour that's needed eg `toggle-class-on-focus`. It doesn't need to be this generic yet, if/when more event actions are needed then this would be useful. For the purpose of this PR I think it obscures the module's purpose.